website/content/docs/builders/alicloud-ecs.mdx

```diff
@@ -3,7 +3,6 @@ description: |
   The `alicloud-ecs` Packer builder plugin provide the capability to build
   customized images based on an existing base images.
 page_title: Alicloud Image Builder
-sidebar_title: Alicloud ECS
```

Comment:
All `.mdx` files have just had their `sidebar_title` value removed from frontmatter, as these values are not longer used.